src/azure/azure-mesh-adapter.ts

```diff
@@ -171,8 +178,21 @@ export class AzureMeshAdapter implements MeshAdapter {
       ...tenantCostInfo.map((tci) => parseFloat(tci.pretaxCost)),
     ].reduce((acc, val) => acc + val);
 
+    let currencySymbol = "";
+    if (tenantCostInfo.length > 0) {
+      currencySymbol = tenantCostInfo[0].currency;
+      for (const tci of tenantCostInfo) {
+        if (tci.currency !== currencySymbol) {
+          throw new MeshAzurePlatformError(
+            AzureErrorCode.AZURE_CLI_GENERAL,
+            `Encoutered two different currency during cost collection for tenant ${tenant.platformTenantId}. This is currently not supported.`,
```

Comment:
```suggestion
            `Encountered two different currency during cost collection for tenant ${tenant.platformTenantId}. This is currently not supported.`,
```

Reply:
Now I'm a bit confused. Why is this error thrown twice in this file?

Reply:
Because we have to ways of collecting cost information. For one we can tell the actual tenant where the problem exists, for the other way its not so easily doable, that's why also the messages differ slightly.